db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/post_provision.rb

```diff
@@ -1 +1,115 @@
+require 'rest_client'
+require 'net/ssh'
 
+def rhsm_user
+  $evm.root['automation_task'].automation_request.options[:attrs][:rhsm_user]
+end
+
+def rhsm_password
+  $evm.root['automation_task'].automation_request.options[:attrs][:rhsm_password]
+end
+
+def rhsm_sku
+  $evm.root['automation_task'].automation_request.options[:attrs][:rhsm_sku]
+end
+
+# need to check if clients need to add pool or just employees
+# need to have activation key command as well
+COMMANDS = [
+  "subscription-manager register --username=#{rhsm_user}  --password=#{rhsm_password}",
+  "subscription-manager attach --pool=#{rhsm_pool_id}",
```

Comment:
we will probably work with SKU and from it take the pool id. an example can be found in rhel_subscribe playbook:
https://github.com/openshift/openshift-ansible/blob/3d72d46b4e5447b80100464da4aa7ac0fa111a3c/roles/rhel_subscribe/tasks/main.yml#L41